apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -102,6 +102,34 @@ export class AppRoomBridge extends RoomBridge {
 		return this.orch.getConverters()?.get('users').convertById(room.u._id);
 	}
 
+	protected async getMessages(
+		roomId: string,
+		options: {
+			limit: number;
+			skip?: number;
+			sort?: Record<string, 1 | -1>;
+		},
+		appId: string,
+	): Promise<IMessage[]> {
+		this.orch.debugLog(`The App ${appId} is getting the messages of the room: "${roomId}"`);
+		const { limit, skip, sort = { ts: -1 } } = options;
+
+		const messageQueryOptions = {
+			limit: Math.min(limit, 100),
+			skip,
+			sort,
+		};
+
+		const messages = await Messages.findVisibleByRoomId(roomId, messageQueryOptions).toArray();
+
+		const messageConverter = this.orch.getConverters()?.get('messages');
+		if (!messageConverter) {
+			throw new Error('Message converter not found');
+		}
```

Comment:
if you are checking for this I'd assume there are chances of this happening? last I remember there wasn't. If required, put this before the model call, let's help the db as much as we can ;)

Reply:
Done, thank you!